Source/UserManagement/Domain/DataCollectors/AggregateRoots/DataCollectorManagement.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using doLittle.Domain;
+using Domain.DataCollectors.Commands;
+using Events.DataCollector;
+
+namespace Domain.DataCollectors.AggregateRoots
```

Comment:
There shouldn't be folders based on technical role.  That is, no AggregateRoots, Validators, CommandHandlers or Commands.  Things should be grouped by Feature.  Where there is too many artifacts, this might indicate that you are missing sub features.